siliconcompiler/remote/client.py

```diff
@@ -665,9 +665,12 @@ def fetch_results(chip, node, results_path=None):
 
     # Note: the server should eventually delete the results as they age out (~8h), but this will
     # give us a brief period to look at failed results.
-    if results_code:
+    if not node and results_code:
```

Comment:
I don't see how this would occur.
If you just want to mute it then I guess completely remove this
and just keep
```
if results_code:
    return
```

Reply:
this condition only applies if no node is specified.

Reply:
I'm just saying I don't see any call that would leave node unspecified